frontend/src/lib/components/chart/chart-page/encoding/chart-encoding/encoding-components/ModelsEncodingDropdown.svelte

```diff
@@ -26,12 +25,10 @@
 	}
 </script>
 
-<EncodingContainer>
-	<Svelecte
-		style="width: 280px; flex:none;"
-		bind:value
-		{options}
-		on:change={valueSelected}
-		renderer={svelecteRenderer}
-	/>
-</EncodingContainer>
+<Svelecte
+	style="width: 280px; flex:none;"
+	bind:value
+	{options}
+	on:change={valueSelected}
+	renderer={svelecteRenderer}
+/>
```

Comment:
Should we add a `container` variable here similar to how we do it for [Metrics](https://github.com/zeno-ml/zeno-hub/blob/e465352c5cbc3d87c1f615cc25d1d2d1adfd2a01/frontend/src/lib/components/chart/chart-page/encoding/chart-encoding/encoding-components/MetricsEncodingDropdown.svelte)?